docs/client/data.js

```diff
@@ -2976,7 +3157,9 @@ DocsData = {
     "summary": "Invoke a method passing an array of arguments."
   },
   "Meteor.call": {
+    "filepath": "ddp/livedata_connection.js",
```

Comment:
Due to the isomorphic nature of Meteor APIs, some methods are defined in different places — with different implementations for each environement. For instance `Meteor.call` is defined in `ddp/livedata_connection.js` but also in `ddp/livedata_server.js` (in this case the implementation is identical but the point still stands, we can imagine having two differents implementations).

Is that something we want to handle? If yes, I guess a solution would be to register an array of locations: `"sourceloc": ["filepath": "ddp/livedata_connection.js", "lineno": 703]` (not sure about the names here).